dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BasicET.java

```diff
@@ -330,26 +330,11 @@ public void testQuayGithubManuallyRegisterDuplicate() {
         @Test
         public void testQuayGithubQuickRegisterWithWDL() {
                 Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file.txt"), "tool", "refresh", "--entry", "quay.io/dockstoretestuser/quayandgithub", "--script" });
+                Client.main(new String[] { "--config", ResourceHelpers.resourceFilePath("config_file.txt"), "tool", "publish", "--entry", "quay.io/dockstoretestuser/quayandgithub", "--script" });
 
                 final CommonTestUtilities.TestingPostgres testingPostgres = getTestingPostgres();
-                final long count = testingPostgres.runSelectStatement("select count(*) from tool where path = 'quay.io/dockstoretestuser/quayandgithub' and validtrigger = 't'", new ScalarHandler<>());
+                final long count = testingPostgres.runSelectStatement("select count(*) from tool where path = 'quay.io/dockstoretestuser/quayandgithub' and ispublished = 't'", new ScalarHandler<>());
                 Assert.assertTrue("the given entry should be valid", count == 1);
-
-                final long count2 = testingPostgres.runSelectStatement("select count(*) from tool, tag, tool_tag where tool.path = 'quay.io/dockstoretestuser/quayandgithub' and tool.id = tool_tag.toolid and tool_tag.toolid = tag.id", new ScalarHandler<>());
-                Assert.assertTrue("the given entry should have three valid tags", count2 == 3);
-        }
-
-        /**
-         * Test adding a entry with an invalid WDL descriptor
-         */
-        @Test
```

Comment:
Hmmm, I think it still might make sense to replace it with a test that checks whether a WDL entry is publishable.